packaging/rpm/flightctl.spec

```diff
@@ -27,6 +30,9 @@ Flightctl Agent is a component of the flightctl tool.
 %prep
 %setup -q -n flightctl-0.0.1
 
+%define source_git_tag $(git describe --tags --exclude latest)
```

Comment:
git is now a dependency looking at the docs we will need to add that to `BuildRequires`[1]?

[1] https://rpm-packaging-guide.github.io/

Reply:
yes

Reply:
it is there already, see here [1]

[1] https://github.com/flightctl/flightctl/blob/main/packaging/rpm/flightctl.spec#L14